### PR TITLE
Fixed Bintray publication issues after all of the updates and bumped version

### PR DESCRIPTION
**Pull request by btkelly**

### CODE REVIEW

File: gandalf/bintray.gradle [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+            publish = true
+            publicDownloadNumbers = true
+            version {
+                desc = libraryDescription
+            }
+        }
+    }
+} else {
+    println 'No Bintray Credentials found, skipping deployment tasks'
+}
```
Comment (stkent) [inline gandalf/bintray.gradle]: How did bintray deployment work before??

Comment (btkelly) [reply gandalf/bintray.gradle]: It pulled in some GitHub gists that contained the deployment stuff. I moved everything into the repo to edit it easier and fix some issues with all of the dependency updates.

Comment (stkent) [reply gandalf/bintray.gradle]: I figured that was the case, I just don't see any deleted gist references in this PR?

**Comment by btkelly** [comment]
Congrats, no :poop: code found in the **gandalf** module as of commit a819df9b4b786777aa2da7120bb995d7d0a90d17!

File: gandalf/bintray.gradle (hunk 1 above)
Comment (btkelly) [reply gandalf/bintray.gradle]: They were deleted in #87

Comment (stkent) [reply gandalf/bintray.gradle]: Ah ok now I'm following :D

**Pull request merged**